### DISCUSSION TITLE
Bazı Oyunların Açılmama Problemine Çözüm Var Mı?

### DISCUSSION THREAD

**Discussion by Slizzle45** [cagritaskn/GoodbyeDPI-Turkey#2] [Q&A] [answered]
Call Of Duty Black Ops 6 gibi oyunlar anti-cheat sisteminden dolayı goodbyedpi açıkken açılmıyor buna çözüm var mıdır?

**Comment by cagritaskn** [comment, marked as answer]
PUBG, CoD Warzone, CoD Black Ops 6 gibi oyunların anti-hile yazılımları GoodbyeDPI'ı hile olarak algılıyor. Bu konuda anti-hile yazılımlarını yaratan şirketlere gerekli raporlar gönderilmiş ancak hala sonuç alınamamış.
İlerleyen zamanlarda umuyorum ki whitelistlerine GoodbyeDPI ve WinDivert kütüphanesi eklenir.

Şimdilik GoodbyeDPI'ı service_remove.cmd ile kaldırmadan bu oyunlara giriş yapabilmek için bilgisayarınızda cmd açıp sırasıyla
```
sc stop GoodbyeDPI
sc stop WinDivert1.4
sc stop WinDivert
```
satırlarını yazıp Enter tuşuna basarak geçici olarak durdurabilirsiniz.
Hizmetler durdurulduktan sonra ve oyundan çıktıktan sonra GoodbyeDPI açıp Discord'a tekrar erişmek için sırasıyla cmd'ye
```
sc start GoodbyeDPI
sc start WinDivert1.4
sc start WinDivert
```
satırlarını yazarak hizmeti tekrar başlatabilirsiniz.

GoodbyeDPI devre dışı iken oyuna girip oyuna girince GoodbyeDPI'ı tekrar aktifleştirerek Discord'a girmeyi deneyebilirsiniz ancak hizmetleri başlattığınızda anti-hile yazılımı sizi oyundan atabilir.
Hizmeti yeniden başlatmadan girebilirsiniz.
Bu sayede hizmeti kaldırmamış olacaksınız ve bilgisayarınızı yeniden başlattığınızda hizmet tekrar çalışıyor olacak
